scripts/license_scanner/package_config.json

```diff
@@ -0,0 +1,11 @@
+{
+    "excluded_directories": [
+        ".git",
+        ".venv",
+        "build",
```

Comment:
recommend to also exclude the `install` folder
what about loading the exclusions from `.gitignore`?

Reply:
I was thinking about it, but need to add logic to add onto it with the excluded directories here. We don't exclude any Python folders in gitignore, for instance.